tests/test_composition.py

```diff
@@ -654,3 +654,77 @@ def test_composition_id_override(stub_broker, do_work):
     assert group_messages[0].options["pipe_target"][0]["options"]["composition_id"] == "id"
     pipeline_messages = pipeline([group([do_work.message(), do_work.message()])]).build(composition_id="id")
     assert all(message.options["composition_id"] == "id" for message in pipeline_messages)
+
+
+@mock.patch("remoulade.brokers.rabbitmq.RabbitmqBroker._get_channel")
+@pytest.mark.confirm_delivery(True)
+@pytest.mark.group_transaction(True)
+def test_pipeline_with_delivery_confirmation_and_group_transaction(
+    mocked_channel, rabbitmq_broker, rabbitmq_worker, stub_result_backend
+):
+    # Given a result backend
+    # And a broker with the results middleware
+    rabbitmq_broker.add_middleware(Results(backend=stub_result_backend))
+
+    # Given an actor that stores results
+    @remoulade.actor(store_results=True)
+    def do_work():
+        return 42
+
+    # Given an actor that stores results
```

Comment:
no really neede to have multiple actors,you can test with only one (maybe use pipe_ignore)